app/assets/javascripts/modules/reset_password.js.coffee

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
```

Comment:
pop up has scroll bars ... what for ? remove them.
make dialog simple !

important thing is: warning message, two buttons: delete, cancel

http://jqueryui.com/dialog/